config/config.sample.yaml

```diff
@@ -1,155 +1,221 @@
   #   # Should sync *all* team users to Matrix
+  #   #
   #   users:
   #     enabled: true
   # Defaults for all other teams
   all:
     channels:
       enabled: false
-    # whitelist: []
-    # blacklist: []
-    # alias_prefix: "slack_"
+      #whitelist: []
+      #blacklist: []
+      #alias_prefix: "slack_"
     users:
       enabled: false
 
+# Optional.
+#
 provisioning:
   enabled: true
+
   # Should the bridge deny users bridging channels to private rooms.
+  #
   require_public_room: true
-  # Should the bridge allow usesr to bridge private channels.
+  
+  # Should the bridge allow users to bridge private channels.
+  #
   allow_private_channels: true
   limits:
     room_count: 20
     team_count: 1
+  
   # Allow deny list for which Slack channels may be bridged.
   # If allow is defined, all other channels are blocked by default
   # If deny is defined, all other channels are allowed by default
   # If both are defined, allow takes precedence.
+  #
   channel_adl:
     allow: 
-      - "CCZ41UJV7"
-      - "#open.*"
+      #- "CCZ41UJV7"
+      #- "#open.*"
     deny:
-      - "CRBCPA771"
-      - "#secret.*"
+      #- "CRBCPA771"
+      #- "#secret.*"
 
+# Optional. Allows Matrix users to optionally authenticate their Slack accounts.
+# When a Matrix user posts, the bridge will post using their Slack account.
+#
 puppeting:
   # Should the bridge allow users to puppet their accounts
+  #
   enabled: false
+  
   # Should the bridge send users a onboarding message when
   # they join a Slack channel for the first time.
+  #
   onboard_users: false
+  
   # Settings regarding who can send direct messages.
   # If allow is defined, all other users are blocked by default
   # If deny is defined, all other users are allowed by default
   # If both are defined, allow takes precedence.
+  #
   direct_messages:
     allow:
       # Which Slack users may DM Matrix users.
+      #
       slack: 
-        - "U0156TG3W48"
+        #- "U0156TG3W48"
+      
       # Which Matrix users may DM Slack users.
+      #
       matrix: 
-        - "@badactor:badhost"
-        - "@.*:badhost"
+        #- "@badactor:badhost"
+        #- "@.*:badhost"
+
     deny:
       # Which Slack users may NOT DM Matrix users.
+      #
       slack: 
-        - "U0156TG3W48"
+        #- "U0156TG3W48"
+      
       # Which Matrix users may NOT DM Slack users.
+      #
       matrix: 
-        - "@badactor:badhost"
-        - "@.*:badhost"
-bot_profile: # Set the bot's profile on startup, optional
-  displayname: Slack Bridge # Set the displayname, optional
-  avatar_url: "mxc://half-shot.uk/ea64c71ee946ca2f61379abefe2c7d977d276fbb" # Set the avatar, optional
+        #- "@badactor:badhost"
+        #- "@.*:badhost"
+
+# Optional. Set the bot's profile on startup
+bot_profile:
+  # Optional. Set the displayname.
+  #
+  displayname: "Slack Bridge"
+  
+  # Optional. Set the avatar.
+  #
+  avatar_url: "mxc://half-shot.uk/ea64c71ee946ca2f61379abefe2c7d977d276fbb"
```

Comment:
We've since added new options to the config, can you double check we have all of them included in this file?

Reply:
I don't see anything in there for the encryption stuff?